src/Components/Components/src/ElementReference.cs

```diff
@@ -23,13 +24,20 @@ namespace Microsoft.AspNetCore.Components
         /// </remarks>
         public string Id { get; }
 
-        public ElementReference(string id)
+        /// <summary>
+        /// Gets the <see cref="IJSRuntime"/> instance used to perform JS interop calls associated with
+        /// this <see cref="ElementReference"/>.
+        /// </summary>
+        public IJSRuntime JSRuntime { get; }
+
+        public ElementReference(string id, IJSRuntime jsRuntime)
```

Comment:
Since this involves adding new API, you will have to regenerate the reference assembly: https://github.com/dotnet/aspnetcore/blob/master/docs/ReferenceAssemblies.md#when-changing-public-api

Reply:
Could you also add a doc comment for the constructor?